### DISCUSSION TITLE
Adding undocumented endpoints

### DISCUSSION THREAD

**Discussion by cbbm142** [ucfopen/canvasapi#521] [Ideas]
Hi,

Would PRs adding API endpoints that are NOT documented in the Canvas API docs be accepted?  Things like adding or listing LTI keys/developer keys are possible through the API, but are not in the Canvas supplied docs.

Thanks

**Comment by Thetwam** [comment]
It would depend on the individual endpoint and how it is implemented, but I would definitely be willing to review and consider adding them. The developer key actions you mention sound very useful.
